mavros/src/plugins/global_position.cpp

```diff
@@ -89,7 +90,7 @@ class GlobalPositionPlugin : public plugin::PluginBase {
 
 		// home position subscriber to set "map" origin
 		// TODO use UAS
-		hp_sub = gp_nh.subscribe("home", 10, &GlobalPositionPlugin::home_position_cb, this);
+		hp_sub = nh.subscribe("/mavros/home_position/home", 10, &GlobalPositionPlugin::home_position_cb, this);
```

Comment:
No! Node mustn't use absolute paths.

Reply:
The Mavlink HOMEPOSITION message is handled by the home_position.cpp plugin. 

![image](https://github.com/mavlink/mavros/assets/23124814/e1f6d8e2-2da4-4632-8be5-c4d84a9a75cf)

This gets published in the topic `/mavros/home_position/home`. 

Is it desired to create a handler in the global_position.cpp plugin for the same Mavlink topic?

Reply:
```cpp
NodeHandle priv_nh("~");

sub = priv_nh.subscribe("home_position/home", ...);
```

Reply:
Or even:
```
NodeHandle hp_nh("~home_position");

sub = hp_nh.subscribe("home", ...);
```